css-brochure-form/styles.css

```diff
@@ -0,0 +1,68 @@
+.container {
+  width: vw20;
```

Comment:
This does not appear to be a valid width setting, you would want `20vw` indicating 20% of the viewport width, although I would avoid ever using it in any real production code. Typically containers are set sizes like pixels